SimG4CMS/Calo/plugins/module.cc

```diff
@@ -12,8 +11,6 @@
 
 typedef ECalSD EcalSensitiveDetector;
 DEFINE_SENSITIVEDETECTOR(EcalSensitiveDetector);
-typedef HCalSD HcalSensitiveDetector;
```

Comment:
@bsunanda , where HCalSD is now?

Reply:
HcalSensitiveDetectorBuilder instantiates HCalSD as HcalSensitiveDetector by supplying all objects needed from EventSetup